### PR TITLE
changed functions to be ES5 compilant

### LINKED ISSUE #49: [Enhancement] - Convert ES6 code to ES5 in a sample app - app.js
**Description of the enhancement:**
The code at https://github.com/freshdesk/marketplace-sample-apps/blob/master/Freshdesk/v2/your_first_app/app/app.js contains arrow functions that are not ES5 compliant

**For which Freshworks product?**
- [x] Freshdesk
- [ ] Freshchat
- [ ] Freshsales
- [ ] Freshservice
- [ ] Freshcaller

**Describe the solution you'd like**
The front-end code has to be ES5 compliant.

**Reason**
As a part of mid-market and enterprise move, we might stumble on legacy browsers. It is good to have client-side code in ES5

### PR DESCRIPTION
**Pull request by trojano31**
resolves  #49

### CODE REVIEW

File: Freshdesk/v2/your_first_app/app/app.js [hunk 1]
```diff
@@ -4,16 +4,16 @@
...
-$(document).ready(() => {
-  app.initialized().then((_client) => { 
+$(document).ready(function() {
+  app.initialized().then(function(_client) {
     const client = _client;
-    client.events.on('app.activated', () => {
+    client.events.on('app.activated', function() {
       getContactData(client);
     });
   }, err);
```
Comment (hemchander23) [inline Freshdesk/v2/your_first_app/app/app.js]: @trojano31 Could you please remove the `err` object? It is not defined elsewhere

**Comment by hemchander23** [comment]
@trojano31 Thanks for the PR. Kindly check the review comments for a minor change. The rest is good.

File: Freshdesk/v2/your_first_app/app/app.js (hunk 1 above)
Comment (hemchander23) [reply Freshdesk/v2/your_first_app/app/app.js]:
```js
 app.initialized().then(function(_client) {
    const client = _client;
    client.events.on('app.activated', function() {
      getContactData(client);
    });
  });
```
should be good without the `err` callback

Comment (trojano31) [reply Freshdesk/v2/your_first_app/app/app.js]: sure!

**Comment by hemchander23** [comment]
Looks good! Thanks :)
Happy Hacktoberfest

**Pull request merged**